evm-tests/jsontests/src/main.rs

```diff
@@ -1,58 +1,217 @@
-use clap::{App, Arg, SubCommand};
+use clap::{arg, command, value_parser, Arg, ArgAction, Command};
+use ethjson::spec::ForkSpec;
 use evm_jsontests::state as statetests;
+use evm_jsontests::state::{TestExecutionResult, VerboseOutput};
 use evm_jsontests::vm as vmtests;
 use std::collections::HashMap;
+use std::fs;
 use std::fs::File;
 use std::io::BufReader;
+use std::path::{Path, PathBuf};
 
+#[allow(clippy::cognitive_complexity)]
 fn main() {
-	let matches = App::new("jsontests")
-		.version("0.1.0")
-		.author("Wei Tang <hi@that.world>")
-		.about("EVM json test utilities")
+	let matches = command!()
```

Comment:
It's funny, but for me: https://github.com/aurora-is-near/aurora-cli-rs/blob/main/src/cli/simple/mod.rs#L26 is great example how it looks really unclear from CLI interaction point of view.
For me the strength of `aurora-cli` is CLI-types. But looking at the structures I can't imagine how real CLI help output will look. I always  confused looking to that in `aurora-cli`. For me, this approach greatly loses the clarity of the arguments view. But it looks beauteful as strcuters and CLI argiments as types representation. A matter of taste, as always.

Reply:
Yes, a matter of taste. The reason I like the proc macros approach is because I think of CLIs as functions and I want to know the types of the inputs I am passing to the function. But the trade-off is needing to know how `clap-derive` will format the Rust enums/structs into command/arg names.

Reply:
Actually, in that particular case, I just added verbose flags for extended output for debugging. Pretty simple case.

By the way, in SputnikVM should work tracing output (with specific feature flags) with `RUST_LOG=info` and other env flag from [log crate](https://docs.rs/log/latest/log/). But it doesn't. Do you have may be any idea?

Reply:
Clap does allow taking command line arguments from env vars.

https://docs.rs/clap/4.5.4/clap/struct.Arg.html#method.env